docs/cli.md

```diff
@@ -6,22 +6,25 @@ Visit [Installation](./installation.md).
 
 
 ## Help
```

Comment:
I think we need to unify whether the flag descriptions in all help are capitalized or lowercase, what do you think? It looks like this would require a code modification in `vatz/cmd`.

Reply:
@kim201212 
I updated all commands explanation to start with upper letter but couldn't update flag's `help` explanation to upper letter because only this case is already built with cobra library function and it starts with lower letter. I don't think I can fix it now unless I update cobra library or find other way to deal with. Let's leave it to later future tasks.

```
 ./vatz help         
Usage:
   [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  init        Initialize VATZ
  plugin      Plugin commands
  start       Start VATZ
  stop        Stop VATZ
  version     VATZ Version

Flags:
      --debug   Enable debug mode on Log
  -h, --help    help for this command
      --trace   Enable trace mode on Log

```

Reply:
LGTM! Perfect!